### ISSUE TITLE
Critical Accessibility Bug: gr.Chatbot and gr.ChatInterface is broken in v4.37.1.

### ISSUE THREAD

**Issue by chigkim** [gradio-app/gradio#8615]
### Describe the bug

Screen reader cannot read the message anymore.
It just says user's message and bot's message without the actual content.
It worked in previous version.
The embedded examples below on the documentation doesn't work either.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

https://www.gradio.app/guides/creating-a-chatbot-fast

https://www.gradio.app/guides/creating-a-custom-chatbot-with-blocks

### Logs

```shell
No log.
```

### System Info

```shell
4.37.1
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
@hannahblair could you take a look into this?
